packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -459,6 +460,14 @@ class SnackBar extends StatefulWidget {
   ///  * <https://material.io/design/components/snackbars.html>
   final Duration duration;
 
+  /// Whether the snack bar will stay or auto-dismiss after timeout.
+  ///
+  /// If true, the snack bar is still there even after the timeout, until the
+  /// user taps the action button or the close icon. If false, the snack bar
+  /// will be dismissed after the timeout. If this is null but the snackbar
```

Comment:
Maybe 
```
If not provided, this is set to true if [action] is not null
```

Reply:
also consider separate each `if` sentence into a new paragraph for readability

Reply:
Done